firestore/angular-rewrite/src/app/submit-review-modal/submit-review-modal.component.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, ViewEncapsulation, inject, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
```

Comment:
nit: format with prettier

Reply:
Added a format script with prettier to my `package.json` and it seems to give the same output for these lines; I think because it's under the 80 col width it's not adjusting the formatting of these two lines. If needed, I can split the imports onto separate lines; i.e.:

```
import { 
Component, 
ViewEncapsulation, 
inject, 
Inject 
} from '@angular/core';
```

Reply:
In this case if the tooling is fine then this is fine - I was expecting a single empty line between external and internal dependencies.

ie) this break on 19 seems like a formatter would remove it (and possibly put @angular/fire ahead of @angular/material to order alphabetically)

Reply:
The separation between internal and external dependencies makes sense, I'll add it in the next commit!